src/xdocs/google_style.xml

```diff
@@ -2435,6 +2435,30 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
```

Comment:
`Overrides` => `Severity Level`.

You are not touching on all overrides, so it is misleading to name this section as such.

Reply:
I thought it will be section for all variables that are not for suppressions.
Do you think we need section for each future variable?

Reply:
You can "override" where the suppression config is located in google.
https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml#L34

If you are going to name this as a global section you need to touch on every aspect, otherwise it is just confusing.

You say this is for "all" variables but then remove suppression variables from this list just because they are suppressions? Imagine this isn't just 2 sections but a list of 100. You say here is one section that lists "all" variables, but you exclude suppressions and suppressions only appears at the bottom after the other 99 items. If someone doesn't both to scan the other sections and only reads the "all", they will think there is no way to override suppressions.

If we make a list of all, then each option should be shorter in description. Similar to https://checkstyle.org/cmdline.html#Command_line_usage .

Reply:
Ok let's have `Severity Level`.

Reply:
done